### ISSUE TITLE
Question about final gfa files from Minigraph-Cactus

### ISSUE THREAD

**Issue by SimonaSecomandi** [ComparativeGenomicsToolkit/cactus#1397]
Hi all, I generated a pangenome with Minigraph-Cactus with this command:

```
 cactus-pangenome 3_MC_all15/js-3_MC_all15 ./scripts/3_MC_all15.seqfile \
--outName $1 \
--outDir $1 \
--reference hap2 hap1 \
--refContigs $(for i in $(seq 39); do printf "chr$i "; done ; echo "chrZ "chrW") \
--filter 3 \
--giraffe clip filter \
--vcf --vcfReference hap2 hap1 \
--viz --odgi --chrom-vg clip filter --chrom-og \
--gbz clip filter full \
--gfa clip full \
--logFile 3_MC_all15/3_MC_all15.log
```
I have these graphs in the main folder:
```
3_MC_all15.full.gfa.gz  
3_MC_all15.gfa.gz  
3_MC_all15.sv.gfa.fa.gz  #1359 
3_MC_all15.sv.gfa.gz
```
The full graph it the one you can use for visualization with odgi, but what is the 3_MC_all15.gfa.gz graph? is this the clipped one?

I'm also wondering what to do if I want to analyze each chr separately. I have the chromosome graphs in the `3_MC_all15.chroms` subfolder, but they are only in `full.og` and `.d3.vg` formats. However, on the MC paper all the analysis (except short-read mapping) were performed on the clipped graph. So should I split the clipped `3_MC_all15.gfa.gz `in chrs? and why this is not done automatically?
And moreover, if I want to run a `vg Giraffe` test on one of the filtered chromosomes, should I convert the `3_MC_all15.chroms/chr.d3.vg` in gfa, or is there another way? can I recover the whole-genome filtered graph from the indexes?

Many thanks!!
Simona

**Comment by glennhickey** [comment]
Yeah, there are 4 types of graphs, the name conventions apply to all files, but here's an example with `.gfa`. 
* `.sv.gfa` : minigraph output -- just SVs and limited path embeddings
* `.gfa` : default/clipped graph -- used for most vg tools
* `.full.vg` : unclipped full graph -- better for odgi
* `.dX.vg` : allele frequency filtered graph. was used for read mapping but has been made obsolete by [haplotype sampling](https://github.com/ComparativeGenomicsToolkit/cactus/blob/master/doc/pangenome.md#haplotype-sampling-instead-of-filtering-new)

So in your `.chroms` output, because you ran with `--chrom-vg clip filter` and `--odgi` (which is the same as `--odgi full`) you will get

* `chr1.vg` : default/clipped graph 
* `chr1.full.og` : unclipped/full graph
* `chr1.dX.vg` : allele frequency filtered graph (obsolete -- see above)
